Makefile

```diff
@@ -1,5 +1,7 @@
+#!/usr/bin/make -f
+
 PACKAGES=$(shell go list ./...)
-OUTPUT?=build/tendermint
+BUILDDIR ?= $(CURDIR)/build
```

Comment:
is this set somewhere?

Reply:
Do you refer to `CURDIR`? If so, `CURDIR` is always set in the `Makefile` execution environment, for more info see https://www.gnu.org/software/make/manual/html_node/Quick-Reference.html